diff --git i/include/configs/pico-imx8mq.h w/include/configs/pico-imx8mq.h
index 3f105ed77a..776d8f4e25 100644
--- i/include/configs/pico-imx8mq.h
+++ w/include/configs/pico-imx8mq.h
@@ -11,7 +11,7 @@
 #include <asm/arch/imx-regs.h>
 #include "imx_env.h"
 
-#define CONFIG_SYS_BOOTM_LEN		(32 * SZ_1M)
+#define CONFIG_SYS_BOOTM_LEN		(64 * SZ_1M)
 
 #define CONFIG_SPL_MAX_SIZE		(148 * 1024)
 #define CONFIG_SYS_MONITOR_LEN		(512 * 1024)
@@ -168,6 +168,8 @@
 			   "fi; " \
 		   "fi;"
 
+#undef CONFIG_EXTRA_ENV_SETTINGS
+
 #define CONFIG_SYS_INIT_RAM_ADDR        0x40000000
 #define CONFIG_SYS_INIT_RAM_SIZE        0x80000
 #define CONFIG_SYS_INIT_SP_OFFSET \
